### PR TITLE
Replace deprecated functions with new ones

### PR DESCRIPTION
**Pull request by NotSoSuper**
```
bot3-7 (err): /usr/local/lib/python3.5/dist-packages/PIL/ImageFont.py:410: DeprecationWarning: decodestring() is a deprecated alias, use decodebytes()
bot3-7 (err):   ''')), Image.open(BytesIO(base64.decodestring(b'''
bot3-7 (err): /usr/local/lib/python3.5/dist-packages/PIL/ImageFont.py:434: DeprecationWarning: decodestring() is a deprecated alias, use decodebytes()
bot3-7 (err):   '''))))
```

Changes proposed in this pull request:

 * replace deprecated functions that actually call and return the new function in the code

### CODE REVIEW

**Comment by NotSoSuper** [comment]
`decodebytes` is actually a function in versions of python 3.1 and higher, just saw that.

**Comment by hugovk** [comment]
Rather than checking the version and using legacy interface decodebytes, can base64.standard_b64decode or base64.standard_b64decode from the modern interface be used?

File: PIL/ImageFont.py [hunk 1]
```diff
@@ -313,9 +313,11 @@ def load_default():
     from io import BytesIO
     import base64
     f = ImageFont()
+    if sys.version_info < (3, 1):
+        base64.decodebytes = base64.decodestring
```
Comment (homm) [inline PIL/ImageFont.py]: We can't monkeypatch the library.

**Comment by hugovk** [comment]
See PR https://github.com/python-pillow/Pillow/pull/2121 using `base64. b64decode`.

**Pull request closed without merging**